src/common/snippets/src/utils/tokenization_utils.cpp

```diff
@@ -408,24 +421,27 @@ std::shared_ptr<ov::snippets::op::Subgraph> tokenize_ordered_nodes(const ov::Nod
             const auto input = node->input(i);
             const auto parent = input.get_source_output().get_node_shared_ptr();
             const auto constant = ov::as_type_ptr<ov::op::v0::Constant>(parent);
-            if (constant && (ov::shape_size(input.get_shape()) == 1 ||
-                             ov::is_type<ov::op::v0::FakeQuantize>(node) ||
+            if (constant && (ov::shape_size(input.get_shape()) == 1 || ov::is_type<ov::op::v0::FakeQuantize>(node) ||
                              op::Subgraph::constant_input_should_be_inside_body(node))) {
                 // If Constant has one consumer - target node, we add Constant to body_inputs
                 // If Constant has several consumers, we should check that all these consumers are inside Subgraph body
-                // and if all of them are inside body, we can explicitly add Constant to the body_inputs, otherwise we should
-                // make a copy and add copy of Constant to body_inputs
-                // For example, this case is especially valid for Transposes nodes
-                //              (several Transposes have the same order so there can be the common Constant with this order)
+                // and if all of them are inside body, we can explicitly add Constant to the body_inputs, otherwise we
+                // should make a copy and add copy of Constant to body_inputs For example, this case is especially valid
+                // for Transposes nodes
+                //              (several Transposes have the same order so there can be the common Constant with this
+                //              order)
```

Comment:
Minor: something goes wrong with this comment.. Can we fix it?

Reply:
Done